Service/MailgunTransport.php

```diff
@@ -202,4 +203,12 @@ protected function getDomain(Swift_Mime_Message $message)
 
         return $this->domain;
     }
+
+    /**
+     * Not used.
```

Comment:
If it's not used it can be removed?

Reply:
It's not used but it's necessary for the class to comply to the interface